src/TokenWrapper.sol

```diff
@@ -79,21 +137,21 @@ contract TokenWrapper is ITokenWrapper, ERC20 {
     /**
      * @notice Wraps the underlying token `value` and mints wrapper tokens into `msg.sender`'s balance.
      * @dev The `totalSupply` of the wrapped token MUST be less than or equal to the underlying token balance of the current contract.
-     * @param value The value to be wrapped.
+     * @param value The `wei` value to be wrapped.
```

Comment:
Not sure I understand the use of `wei` here. I think we should stick to `wad`

Reply:
Yeah, that confuses me a bit TBH.
The way I built my mental model around it is that `wad` means `10^18`. Since the underlying token has `0` decimals, it didn't sound right in my head to call it `wad`, but I'm not sure about this.